### ISSUE TITLE
Added you as collaborator

### ISSUE THREAD

**Issue by dlutz52** [wardi/cpu#2]
Ian --
I have added you as collaborator to two of my private projects. I am new to this aspect of GitHub so I am not sure this is the correct approach for this. You can contact me via Google Mail as dlutz52 for more info as to what I am up to here.

**Comment by wardi** [comment]
Looks really cool @dlutz52.

As an open-source guy I default to sharing things in public instead of managing invites.

If you want a friendlier URL for the video you could link to my blog post: https://excess.org/lcd2004-tutorial/

**Comment by dlutz52** [comment]
I agree with making things public. I just did not want to do so before you
saw it in case you had any objections to my using your name.
I have updated the silkscreen for the the HD4470 Tutorial Board to use your
website's URL.
You only mentioned the HD4470 Tutorial Board. Did you also see the KiCad
project for the Homebrew CPU series?
I went to update the Homebrew CPU silkscreen and noticed that the title on
your web page for this project read "20xt6 Homebrew CPU Project". What does
the t6 mean? Is that intentional? Just want to verify before I use it in
the silkscreen.
If you have such interest, I can ship you a finished board for each
project--you just provide your own chips and place them in the sockets on
the boards.Just need a mailing address.
That way you can verify that the boards work as expected. After that I can
make the repos public. Just do not want hardware to go public that is
objectionable in any way.
The sockets will be the cheaper type for the 555/74LS00/74LS163 and the
Juried Engineering (of the type you used in the Homebrew CPU video) for
the SST39SF040 chips since they will be changed out more often. Another
option is for the 555/74LS00/74LS163 chips to just be soldered to the board
itself.
Let me know if you want me to send you boards.
I will use PCBWay for the manufacturing and will look into leaving the
boards on their website as a project that can be order straight from PCBWay.

David

**Comment by wardi** [comment]
"20xt6" is inspired by https://homestarrunner.com/stinkogame but has a deeper meaning to be revealed at a future date :-)

I've never designed a PCB before but it looks like you have a PCB for maybe the "Milestone 1 complete" plus a connection for the halt?

You could reference the stage of the project that is being recreated in PCB form on the design. Assuming I start publishing videos again there will be a number of interesting stages along the way to https://www.youtube.com/shorts/fGyHHWiAYDM and beyond with source code I'll put on my web site.

Thanks for sharing this!

**Comment by dlutz52** [comment]
Yes to "a PCB for maybe the "Milestone 1 complete" plus a connection for
the halt?"
Halt was in video 7 of playlist before Milestone 1.
I added changes to the README based on my understanding of what you wrote
in your reply email -- and quite a few more.
Check it out: https://github.com/dlutz52/Homebrew-CPU-PCB.
Let me know if any changes are needed.
I also added PDF files of the schema for both projects for any non-Kicad
users.
I have had a quick look at Discrete Logic Rhythm Game Prototype (no CPU)
<https://www.youtube.com/watch?v=yo5xX4GLc_U> and my first impression is
that I might be able to accomodate the keyboard and health meter as
"daughter boards" to the current board - separate PCBs using header pins
and cables(?) to connect, kind of modular. The only thing that draws my
attention with respect to the existing board is the 16 pin IC to the right
of the LCD. Any schematics on any of this new stuff?

David
